spec/fabricators/keyword_fabricator.rb

```diff
@@ -3,6 +3,7 @@
 Fabricator(:keyword) do
   content { FFaker::Lorem.characters(16) }
   status :processing
+  user
```

Comment:
Would this do the same as the following:

```suggestion
  user { Fabricate(:user) }
```